XamarinCommunityToolkit/Views/SideMenuPosition.shared.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace Microsoft.Toolkit.Xamarin.Forms.UI.Views
+{
+    public enum SideMenuPosition
+    {
+        None = 0,
+        LeftMenu = 1,
+        RightMenu = -1
```

Comment:
Is there any specific reason for this to be -1?
(also for the state)

Reply:
For this control -- not